features/step_definitions/account_creation_steps.rb

```diff
@@ -0,0 +1,45 @@
+Given(/^I do not have an account on the site$/) do
+  User.find_by_email('my_email@example.com') == nil
```

Comment:
1. Don't use `find_by_<attribute>`. Use `find_by(<attribute>: ...)` instead
2. Here you're testing that you don't have an account. Instead, do `User.find_by(email: 'my_email@example.com').delete_all`